UiPath.Ipc.net6.0-windows.received.txt

```diff
@@ -0,0 +1,221 @@
+        public static System.IO.Pipes.PipeSecurity Deny(this System.IO.Pipes.PipeSecurity pipeSecurity, System.Security.Principal.IdentityReference sid, System.IO.Pipes.PipeAccessRights pipeAccessRights) { }
+        public static System.IO.Pipes.PipeSecurity Deny(this System.IO.Pipes.PipeSecurity pipeSecurity, System.Security.Principal.WellKnownSidType sid, System.IO.Pipes.PipeAccessRights pipeAccessRights) { }
+        public static System.IO.Pipes.PipeSecurity LocalOnly(this System.IO.Pipes.PipeSecurity pipeSecurity) { }
+        [System.ComponentModel.Browsable(false)]
+        public static bool PipeExists(string pipeName, int timeout = 1) { }
+    }
+    public abstract class IpcBase
+    {
+        protected IpcBase() { }
+        public System.TimeSpan RequestTimeout { get; set; }
+        public System.Threading.Tasks.TaskScheduler? Scheduler { get; set; }
+        public System.IServiceProvider? ServiceProvider { get; set; }
+    }
+    public sealed class IpcClient : UiPath.Ipc.IpcBase
+    {
+        [System.Obsolete("Constructors of types with required members are not supported in this version of " +
+            "your compiler.", true)]
+        public IpcClient() { }
+        public System.Func<System.Threading.CancellationToken, System.Threading.Tasks.Task>? BeforeConnect { get; set; }
+        public System.Func<UiPath.Ipc.CallInfo, System.Threading.CancellationToken, System.Threading.Tasks.Task>? BeforeOutgoingCall { get; set; }
+        public UiPath.Ipc.EndpointCollection? Callbacks { get; set; }
+        public Microsoft.Extensions.Logging.ILogger? Logger { get; init; }
+        public UiPath.Ipc.ClientTransport Transport { get; init; }
+        public TProxy GetProxy<TProxy>()
+            where TProxy :  class { }
+    }
+    public class IpcProxy : System.Reflection.DispatchProxy, System.IDisposable
+    {
+        public IpcProxy() { }
+        public System.IO.Stream? Network { get; }
+        public event System.EventHandler ConnectionClosed;
+        public System.Threading.Tasks.ValueTask CloseConnection() { }
+        public void Dispose() { }
+        protected override object? Invoke(System.Reflection.MethodInfo? targetMethod, object?[]? args) { }
+    }
+    public sealed class IpcServer : UiPath.Ipc.IpcBase, System.IAsyncDisposable
+    {
+        [System.Obsolete("Constructors of types with required members are not supported in this version of " +
+            "your compiler.", true)]
+        public IpcServer() { }
+        public UiPath.Ipc.EndpointCollection Endpoints { get; init; }
+        public UiPath.Ipc.ServerTransport Transport { get; init; }
+        public System.Threading.Tasks.ValueTask DisposeAsync() { }
+        [System.Diagnostics.CodeAnalysis.MemberNotNull(new string[] {
+                "Transport",
+                "_accepter"})]
+        public void Start() { }
+        public System.Threading.Tasks.Task WaitForStart() { }
+        public System.Threading.Tasks.Task WaitForStop() { }
```

Comment:
what;s the differentce between WaitForStop() and DisposeAsync()?
I'd like to avoid supporting the "restart" scenario.